torch/csrc/CudaIPCTypes.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <TH/TH.h>
+
+struct CudaIPCReceivedData {
```

Comment:
docs!

Reply:
It would also be nice to just have something like `Note [Distributed CUDA tensor refcount]` and then text below which explains the general strategy for how the refcounting works; it'll help future readers get oriented more quickly.